src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -381,6 +389,37 @@ def build_cmd(self, env, host, **kwargs):
 
         return tst_cmd
 
+    def convert_xml(self, xml_file):
+        """Modify the xml file"""
+
+        fd = open(xml_file, 'r')
+        ofd = open('{}.xml'.format(xml_file), 'w')
+        for line in fd:
+            if self.src_dir in line:
+                L = re.sub('<dir>\/*' + self.src_dir + '\/*', '<dir>', line)
```

Comment:
(lint) '\/'. String constant might be missing an r prefix.
(lint) '\/'. String constant might be missing an r prefix.